src/components/_classes/input/Input.js

```diff
@@ -22,6 +22,7 @@ export default class Input extends Multivalue {
       name: this.options.name,
       type: this.component.inputType || 'text',
       class: 'form-control',
+      id: this.component.key,
```

Comment:
We've tried hard to stay away from using ids. This is because The same form can be on a page multiple times and different forms on the same page could have the same keys for components. The label should work with the "name" attribute of the component, not the id.

Reply:
Thanks for the response @randallknutson 
Could we then somehow use label "for" with the "name" attribute, and set component's id same as its name?
It is a pretty important requirement for form accessibility to have the the label & input linked up.

Reply:
Maybe you can use ```id:this.options.name``` and change label for to that id?

Reply:
You don't need to set the id in order to set the label and input to be linked. If the label "for" is the same as the input "name" it will work correctly. We would definitely accept this as a pull request.

Reply:
We have a pretty big project working on this right now.  https://github.com/formio/formio.js/pull/1732

Reply:
I would vote to close this in favor of the changes being made to https://github.com/formio/formio.js/pull/1732